nipype/interfaces/afni/preprocess.py

```diff
@@ -438,6 +438,12 @@ class AllineateInputSpec(AFNICommandInputSpec):
         traits.Enum(*_dirs),
         argstr='-nwarp_fixdep%s',
         desc='To fix non-linear warp dependency along directions.')
+    verb = traits.Bool(
```

Comment:
Feel free to say no, but would this make more sense as "verbose" instead of "verb"? I think that'll make interfaces a little friendlier to new users, while experienced AFNI users would know "verb" means "verbose", so it shouldn't cause confusion.

Reply:
yes I agree, I was hesitating on this one. Is that OK if I change all the AFNI `verb` (even introduced before my changes) to `verbose` ?

Reply:
Hmm. No, that would require deprecating, because somebody depends on this behavior. If it's consistent with existing interfaces, `verb` seems fine.

Reply:
Ok